source/onboard/mattermost-limits.rst

```diff
@@ -14,19 +14,23 @@ In addition to these limits, visit our pricing page to see a complete list of fe
 
    These limits do not apply to self-hosted deployments. 
 
-Mattermost Free limits
-----------------------
+Mattermost Cloud Free limits
+----------------------------
 
 **Platform limits**
 
-- Maximum 1 team. Any additional teams created during a trial or on a paid plan will be archived.
-- 10 GB file storage across the platform, with 100 MB upload limit. The oldest files that exceed this limit will be archived.
 - Unlimited plugins can be installed but only 5 can be enabled at one time. Core plugins such as Boards, Calls, and Playbooks do not count toward this limit.
+- 1 GB file storage across the platform, with 100 MB upload limit. The oldest files that exceed this limit will be archived.
+- Maximum 1 team. Any additional teams created during a trial or on a paid plan will be archived.
 
 **Channels limits**
 
 - 10,000 message history limit. The oldest messages that exceed this limit will be archived.
 
+**Calls limits**
+
+- Up to 8 participants per channel can join calls that last up to 40 minutes.
+
 **Boards limits**
```

Comment:
@cwarnermm All Boards limits will be removed for v7.7, so there will no longer be any limits of:
- Maximum 5 saved views per board.
- Only the 500 most recently updated cards on the server are displayed.

This entire section can be removed for the next release. Thanks!

Reply:
Thanks, @wuwinson! I've removed the Boards limits from the docs page in a new commit.